packages/subgraph/package.json

```diff
@@ -7,6 +7,7 @@
   "scripts": {
     "lint": "eslint . --ext .ts",
     "build:contracts": "cd ../contracts && yarn build",
+    "build:contracts-versions": "cd ../contracts-versions && yarn build",
```

Comment:
Does subgraph need older versions?

Reply:
no but since the contracts needs contracts-versions, now subgraph also need to build contracts-versions

Reply:
If so, shouldn't we simply have a script so that subgraph "builds" contracts... and then contracts "build" whatever else it needs?

I would avoid this type of build coupling, if possible